template.yaml

```diff
@@ -332,7 +337,13 @@ Globals:
                 !Ref Environment,
                 dynatraceSecretArn,
               ]
-        DT_OPEN_TELEMETRY_ENABLE_INTEGRATION: "true"
+        DT_OPEN_TELEMETRY_ENABLE_INTEGRATION:
+          !FindInMap [
+            EnvironmentConfiguration,
+            !Ref Environment,
+            opentelemetryEnabled,
+          ]
+        OTEL_INSTRUMENTATION_AWS_SDK_EXPERIMENTAL_USE_PROPAGATOR_FOR_MESSAGING: "true"
```

Comment:
I assume it's fine to set this to true for lambdas that don't have OT turned on?